src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -496,15 +500,34 @@ var DynamicTable = new Class({
         this.selectedRows.empty();
     },
 
-    selectRow: function(rowId) {
-        this.deselectAll();
+    selectRow: function(rowId, clear) {
+        if (clear)
+            this.deselectAll();
         this.selectedRows.push(rowId);
+        this.setRowClass();
+        this.onSelectedRowChanged();
+    },
+
+    deselectRow: function(rowId) {
+        this.selectedRows.erase(rowId);
+        this.setRowClass();
+        this.onSelectedRowChanged();
+    },
+
+    selectRows: function(rowIdStart, rowIdEnd) {
```

Comment:
I think you need an `assert(rowIdStart <= rowIdEnd)` or something equivalent.

Reply:
This is meant  to work between any two rows, regardless of direction.

Reply:
>This is meant to work between any two rows, regardless of direction.

Then you should name them `rowId1, rowId2`.